githubutil/auth.go

```diff
@@ -129,10 +129,15 @@ func (a GitHubAppAuther) InsertHTTPAuth(req *http.Request) error {
 
 func (a GitHubAppAuther) GetIdentity() (string, error) {
 	ctx := context.Background()
-	client, err := NewInstallationClient(ctx, a.ClientID, a.InstallationID, a.PrivateKey)
```

Comment:
It doesn't look like `NewInstallationClient` is actually used outside the package, and don't need to be exported. Same with hypothetical `NewAppClient`, and `NewClient` can also go away once all usage is migrated.

Reply:
I've changed `NewInstallationClient` to `newInstallationClient`